masonite/view.py

```diff
@@ -51,3 +68,76 @@ def extend(self):
     def share(self, dictionary):
         self.dictionary.update(dictionary)
         return self
+
+    def cache_for(self, time=None, type=None):
+        """
+        Set time and type for cache
+        """
+        self.cache = True
+        self.cache_time = float(time)
+        self.cache_type = type
+        if self.__is_expired_cache():
+            self.__create_cache_template(self.template)
+        return self
+
+    def __create_cache_template(self, template):
+        """
+        Save in the cache the template
+        """
+
+        cache_type = self.cache_type.lower()
+        calc = 0
+        if cache_type == "second" or cache_type == "seconds":
+            # Set time now for
+            calc = 1
+        elif cache_type == "minutes" or cache_type == "minute":
+            calc = 60
+        elif cache_type == "hours" or cache_type == 'hour':
+            calc = 60 * 60
+        elif cache_type == "days" or cache_type == 'day':
+            calc = 60 * 60 * 60
+        elif cache_type == "months" or cache_type == 'month':
+            calc = 60 * 60 * 60 * 60
+        elif cache_type == "years" or cache_type == 'year':
+            calc = 60 * 60 * 60 * 60 * 60
+        else:
+            # If is forever
+            return True
```

Comment:
This shouldn't return True, it should be `raise ValueError('{0} is not a valid caching type.'.format(cache_type))`